### PR TITLE
Composite tags

### PR DESCRIPTION
**Pull request by free6k**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -4,3 +4,5 @@ coverage
 tests/runtime
 coverage.xml
 .vagrant
+.idea
+.DS_STORE
```
Comment (Philosoft) [inline .gitignore:8]: os and env-specific files. add this lines to your global `.gitignore` and be happier next year

Comment (free6k) [inline .gitignore:8]: You too :) what i must do next, that you added my pull request? ) rewrite .gitignore?

**Comment by Philosoft** [comment]
why close PR? just add new commit

**Comment by Philosoft** [comment]
Can you provide use-case for that type of caching?

**Comment by free6k** [comment]
I am noob, I can not do so)))) i create new pull-request))) i updated Readme.md )

**Pull request closed without merging**